Description: Scoreboard handle in daemon mode should be set to NULL for
 Apache 2.4 to avoid crash in lingering close.
Origin: upstream, https://code.google.com/p/modwsgi/source/detail?r=20979c8abd2f722d210d419b96c85e46a36e2d75
Bug-Debian: http://bugs.debian.org/719304

--- a/mod_wsgi.c
+++ b/mod_wsgi.c
@@ -10599,7 +10599,13 @@
      * will add their own input/output filters to the chain.
      */
 
+#if AP_MODULE_MAGIC_AT_LEAST(20110619,0)
+    /* For 2.4 a NULL sbh pointer should work. */
+    sbh = NULL;
+#else
+    /* For 2.2 a dummy sbh pointer is needed. */
     ap_create_sb_handle(&sbh, p, -1, 0);
+#endif
 
     c = (conn_rec *)apr_pcalloc(p, sizeof(conn_rec));
 
